src/coreclr/src/debug/createdump/createdump.h

```diff
@@ -65,4 +65,44 @@ typedef int T_CONTEXT;
 #include "threadinfo.h"
 #include "memoryregion.h"
 #include "crashinfo.h"
+
+#ifdef TARGET_ANDROID
+// copied from .tools/android-rootfs/android-ndk-r21/sysroot/usr/include/linux/elfcore.h
```

Comment:
Is there any license on this?

Reply:
On top of this header, it says:

```c
/****************************************************************************
 ****************************************************************************
 ***
 ***   This header was automatically generated from a Linux kernel header
 ***   of the same name, to make information necessary for userspace to
 ***   call into the kernel available to libc.  It contains only constants,
 ***   structures, and macros generated from the original header, and thus,
 ***   contains no copyrightable information.
 ***
 ***   To edit the content of this header, modify the corresponding
 ***   source file (e.g. under external/kernel-headers/original/) then
 ***   run bionic/libc/kernel/tools/update_all.py
 ***
 ***   Any manual change here will be lost the next time this script will
 ***   be run. You've been warned!
 ***
 ****************************************************************************
 ****************************************************************************/
```

The NDK itself does not include `kernel-header` directory or `update_all.py` file.

Reply:
Could you put this into a separate file and include the whole thing? I think this will need more work do properly wrt. licensing.

Reply:
Does the createdump tool actually work with this? Just want to make sure that it is really going to be needed.

If you are not able to verify that createdump works, I would exclude it from build on Android for now.

Reply:
Thank you. I will disable createdump tool compilation on Android and revert the related changes for now; since we already have evidence that there are some issues found during the device testing: https://github.com/dotnet/runtime/issues/33237#issuecomment-597507650 (I haven't gotten that far, but intend to investigate further after this round of changes). We can always enable createdump back and use the separate header approach, once the runtime gets functional on the device.

Reply:
Reverted all changes in createdump directory and skip the directory inclusion from root cmake lists file.